src/main/SnakeAndLadders/Practice/Model/Ladders.java

```diff
@@ -0,0 +1,27 @@
+package SnakeAndLadders.Practice.Model;
+
+public class Ladders {
+    private int start_pos;
```

Comment:
Avoid using both camelCase and snake_case in the code. Better to use camelCase only in case of Java.